firmware/src/main.cpp

```diff
@@ -208,7 +212,48 @@ void testModeLoop() {
   }
 }
 
+unsigned long lastPauseTime;
+bool beepPaused = false;
+uint8_t beepCount = 0;
+bool beepState = false;
+void lowVoltageBeep() {
+  if (!beepPaused) {
+    if (!buzzer.isPlaying()) {
+      if (beepState) {
+        buzzer.playFrequency(440, 250, 7);
+      }
+      else {
+        buzzer.playFrequency(220, 250, 7);
+      }
+      beepState = !beepState;
+      beepCount++;
+    }
+
+    if (beepCount > 10) {
+      beepCount = 0;
+      beepPaused = true;
+      lastPauseTime = millis();
+    }
+  }
+  else {
+    if (millis() - lastPauseTime > 3000) {
+      beepPaused = false;
+    }
+  }
+}
```

Comment:
Can you use `play()` in `PLAY_CHECK` mode so we do not need to handle this logic? Our code becomes something like:

```
if brownout:
  if not playCheck():  
    play("whatever seq of notes you want")
```